Extension/src/LanguageServer/ui_new.ts

```diff
@@ -414,24 +415,17 @@ export class NewUI implements UI {
         }
     }
 
-    private compilerTimout?: NodeJS.Timeout;
-    public async showCompilerStatusIcon(show: boolean): Promise<void> {
-        if (!telemetry.showStatusBarIntelliSenseIndicator()) {
+    public async showConfigureIntelliSenseStatusIcon(show: boolean, client?: Client): Promise<void> {
```

Comment:
Same here, I would think "button" was a better description of what we're using over "StatusIcon"